app/Domain/Integrations/IntegrationType.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Integrations;
+
+enum IntegrationType: string
+{
+    case EntryApi = 'entry-api';
+    case SearchApi = 'search-api';
+    case UitpasApi = 'uitpas-api';
```

Comment:
UiTPAS won't be included in the first phase if I remember correctly, so maybe best to leave this out for now?

Reply:
Agreed! Will do that.